core/src/main/java/com/linecorp/armeria/server/http/auth/OAuth2Token.java

```diff
@@ -0,0 +1,56 @@
+package com.linecorp.armeria.server.http.auth;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Objects;
+
+/**
+ * The bearer token of <a href="https://tools.ietf.org/html/rfc6750">OAuth 2.0 authentication</a>.
+ * */
+public final class OAuth2Token {
+
+    /**
+     * Creates a new {@link OAuth2Token} from given {@code accessToken}.
+     * */
+    public static OAuth2Token of(String accessToken) {
+        return new OAuth2Token(accessToken);
+    }
+
+    /**
+     * Access token.
+     * */
```

Comment:
- `* */` -> `*/`
- Please consider removing the private field comments that do not add information. Everyone knows it's an access token field from its name.